gulpfile.babel.js

```diff
@@ -66,5 +66,5 @@ gulp.task('default', (done) => {
   if (args.production)
     runSequence('server', done);
   else
-    runSequence('server', 'karma-dev', done);
+    runSequence('server', done);
```

Comment:
I'll remove this change from the PR before merging :smile: 

It opens up new browser window and somehow OS X thinks it's the primary browser window and opens all links in it.